spec/policies/division_policy_spec.rb

```diff
@@ -10,27 +10,41 @@
   context 'being a member of a division' do
     let(:user) { create(:user, :member, division: division) }
 
-    permit_actions [:index, :show]
+    it 'can not index' do
+      expect(described_class.new(user, Division).index?).to be_falsey
+    end
```

Comment:
Have you tried just putting this in `forbid_actions`? It should still work I think. Same below.